lib/zenaton/services/graphql.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require 'net/http'
+require 'zenaton/exceptions'
+
+module Zenaton
+  # Collection of utility classes for the Zenaton library
+  module Services
+    # Wrapper class around HTTParty that:
+    # - handles graphql calls
+    # - translates exceptions into Zenaton specific ones
+    # rubocop:disable Metrics/ClassLength
+    class GraphQL
+      # Net::HTTP errors translated into a Zenaton::ConnectionError
+      ALL_NET_HTTP_ERRORS = [
```

Comment:
Can't we reuse the existing http client instead of duplicating this logic here?

Reply:
The error management is not the same.